content/posts/newsletter/2020-05-01.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Draft Title - Date
+save_as: tba
+url: tba
+date: 2020-05-01 00:00
+author: emus
+blog_id: newsletter
+category: Newsletter
+---
+
+# XMPP@home - 3 June 2020
+
+## Intro
+
+Welcome to the XMPP newsletter covering the month of May 2020.
+
```

Comment:
## Help us, help you build the newsletter

Reply:
Help us, help you ... ?

Reply:
Yeah definitely not the best sentence: I wanted to show that we (the writers of the newsletter) need help, and you (projet leader or event organiser) need promotion... :-/